src/context.rs

```diff
@@ -913,7 +935,18 @@ impl Context {
         };
         let ret = Error::from_tss_rc(ret);
         if ret.is_success() {
-            Ok(ObjectHandle::from(esys_object_handle))
+            let new_object_handle = ObjectHandle::from(esys_object_handle);
+            // If you look at the specification and see that it says ESYS_TR_NULL
+            // then that is an error in the spec. ESYS_TR_NULL was renamed to
+            // ESYS_TR NONE.
+            if new_object_handle != ObjectHandle::None {
```

Comment:
Maybe instead of this and the other check that also checks against `Null`, just add an
```
impl ObjectHandle {
    pub(crate) is_none() -> bool ...
```
?